tests/unittests/onnxImporterTest.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "ImporterTestUtils.h"
+#include "glow/ExecutionEngine/ExecutionEngine.h"
+#include "glow/Graph/Graph.h"
+#include "glow/Importer/ONNX.h"
+#include "gtest/gtest.h"
+
+using namespace glow;
+
+/// Test loading conv op from a ONNX model.
+/// The input is N*C*H*W (1*1*3*3), the kernels is {2, 2},
+/// strides is {1, 1}, pads is {1, 1, 1, 1}, group is 1.
+TEST(onnx, importConv) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetFilename("tests/models/onnxModels/model.onnxtxt");
+
+  SaveNode *output;
+  // Destroy the loader after the graph is loaded since the following execution
+  // will not depend on anyting from the loader.
+  {
+    Tensor data;
+    getImageData(&data, 1, 1, 3, 3);
+    ONNXModelLoader onnxLD(NetFilename, {"data"}, {&data}, *F);
+    output = onnxLD.getSingleOutput();
```

Comment:
Could you check the shape of F as well to make sure it matches what is in the model.onnxtxt?

Reply:
I think since the model is run without error messages and we get the correct result, it should prove our loader's correctness.

Reply:
That's another aspect. Although unlikely, you could imagine that we load incorrectly while still producing the correct result.

What I am saying is checking the results are correct is one dimension, the other is to check the loader produces a graph with the expected property like the number of created variables, the number of convolutions, how things are connected together, etc.